src/main/flight/pid.c

```diff
@@ -641,21 +630,23 @@ static void detectAndSetCrashRecovery(
 {
     // if crash recovery is on and accelerometer enabled and there is no gyro overflow, then check for a crash
     // no point in trying to recover if the crash is so severe that the gyro overflows
-    if ((crash_recovery || FLIGHT_MODE(GPS_RESCUE_MODE)) && !gyroOverflowDetected()) {
-        if (ARMING_FLAG(ARMED)) {
-            if (getMotorMixRange() >= 1.0f && !inCrashRecoveryMode
-                && ABS(delta) > crashDtermThreshold
-                && ABS(errorRate) > crashGyroThreshold
-                && ABS(getSetpointRate(axis)) < crashSetpointThreshold) {
+    if ((crash_recovery || FLIGHT_MODE(GPS_RESCUE_MODE)) && !gyroOverflowDetected())
+    {
+        if (ARMING_FLAG(ARMED))
+        {
+            if (getMotorMixRange() >= 1.0f && !inCrashRecoveryMode && ABS(delta) > crashDtermThreshold && ABS(errorRate) > crashGyroThreshold && ABS(getSetpointRate(axis)) < crashSetpointThreshold)
+            {
```

Comment:
not sure if it is worth using this format for long if statements. Simply as the rest of the code uses the previous formatting.

Reply:
TBH I do not like autoformatters just because for different things. I wanted to fix formatting manually but gave up after 20 minutes. There were so many formatting problems that fixing this manually was over any reasonable time budget.
Good catch, well split this too